pkg/controller/webhook.go

```diff
@@ -43,7 +44,13 @@ func callWebhook(webhook string, payload interface{}, timeout string) error {
 		return err
 	}
 
-	req, err := http.NewRequest("POST", hook.String(), bytes.NewBuffer(payloadBin))
+	httpClient := retryablehttp.NewClient()
+	httpClient.RetryWaitMin = 5 * time.Second
```

Comment:
is there any particular reason why we aren't just using the default?

Reply:
I was just using the same setting as https://github.com/fluxcd/pkg/blob/main/http/fetch/archive_fetcher.go#L51 I can just remove these values to just use defaults if preferred.

Reply:
okay thats fair. i'm thinking about how we can make the retry behavior predictable for users. if the analysis interval is 15s and the number of retries of 4, the scheduler will move to the next step before retrying for the 4th time. i guess we could advertise this retry interval in the documentation for this field, so that users have some advice when specifying the interval and the number of retries?

Reply:
actually, since its a blocking call it should be fine, ignore the above comment :)